Mathlib/Tactic/Linter/RefineLinter.lean

```diff
@@ -23,15 +23,15 @@ This linter is an incentive to discourage uses of `refine'`, without being a ban
 
 open Lean Elab
 
-namespace Mathlib.Linter
+namespace Mathlib.Linter.Style
 
 /-- The "refine" linter flags usages of the `refine'` tactic.
 
 The tactics `refine` and `refine'` are similar, but they handle meta-variables slightly differently.
 This means that they are not completely interchangeable, nor can one completely replace the other.
 However, `refine` is more readable and (heuristically) tends to be more efficient on average.
 -/
-register_option linter.refine : Bool := {
+register_option linter.style.refine : Bool := {
```

Comment:
I *think* the other PRs put the `register_option` outside of the style namespace. I'm not sure if that's a meaningful detail.

Reply:
I'll just leave it as you suggested.